modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,289 @@
+                - "--disable_storing_incomplete_runs=true"
+                name: watcher
+                resources: {}
+#...
+----
++
+If set to `true`, runs are not stored until they are complete, improving system performance. If set to `false`, runs are stored upon creation and continuously upserted whenever a change occurs. This is the current default behavior. The `true` setting is planned to become the default in a future release.
+
+Optimization to skip processing of already stored PipelineRuns::
+* {tekton-results} skips processing of already stored `PipelineRuns`, improving performance and reducing API server calls. This optimization enhances overall efficiency and responsiveness, aiming to reduce server load and conserve resources.
+
+.{pac}
+
+Support for JSON body in incoming webhooks::
+* {pac} supports passing incoming webhook parameters in the JSON POST body, while maintaining compatibility with URL query parameters:
++
+[source, terminal]
+----
+curl -H "Content-Type: application/json" -X POST \
+"https://control.pac.url/incoming" -d '{"repository":"repo","branch":"main", \
+"pipelinerun":"target-pipelinerun","secret":"very-secure-shared-secret"}'
+----
++
+Using the request body for sensitive information reduces exposure in logs and improves security. For more details, see link:https://pipelinesascode.com/docs/guide/incoming_webhook/#recommended-post-json-body-method[Recommended (POST JSON body) method].
+
+Detailed logging for GitHub API calls::
+* {pac} supports detailed logging for GitHub API calls, providing insights into API interactions, durations, and rate-limiting. By setting the controller log level to 'debug', you can troubleshoot complex issues more efficiently. This enhancement refactors code to instrument GitHub calls and affects all types of GitHub API calls within the provider. For more details, see link:https://pipelinesascode.com/docs/install/logging/#debugging-api-interactions[Debugging API interactions].
+
+New auto-configure-repo-repository-template option::
+* {pac} adds the `auto-configure-repo-repository-template` setting, a new option in `auto-configure-new-github-repo` feature setting to consolidate repository creation within a single namespace, streamlining management for those with multiple repositories. For more details, see "Additional resources".
+
+Support for relative task references in remote Pipeline definitions::
+* You can use relative paths to reference tasks from within a remote `Pipeline` definition. The {pac} resolver automatically builds the full URL for the task based on the location of the remote `PipelineSpec` definition. This improves pipeline organization and portability by keeping tasks and pipelines co-located, which is especially useful when working with tagged releases or branches. For more details, see link:https://pipelinesascode.com/docs/guide/resolver/#overriding-tasks-from-a-remote-pipeline-on-a-pipelinerun[Overriding tasks from a remote pipeline on a PipelineRun].
```

Comment:
You can now use relative paths to reference tasks within a remote `Pipeline` definition. The {pac} resolver automatically constructs/builds the entire/complete URL for the task based on the location of the remote `PipelineSpec` definition. This enhancement improves pipeline organization and portability, particularly when working with tagged releases or branches. For more information, see link:https://pipelinesascode.com/docs/guide/resolver/#overriding-tasks-from-a-remote-pipeline-on-a-pipelinerun[Overriding tasks from a remote pipeline on a PipelineRun].